sdk/src/test/scala/com/horizen/account/wallet/AccountWalletTest.scala

```diff
@@ -195,4 +191,98 @@ class AccountWalletTest
     assertEquals("SidechainWallet different exception expected during removing new Secret.", expectedException, failureResult.failed.get)
   }
 
+  @Test
+  def testGenerateNextSecret(): Unit = {
+    val mockedSecretStorage: SidechainSecretStorage = mock[SidechainSecretStorage]
+
+    val accountWallet = new AccountWallet(
+      "seed".getBytes(),
+      mockedSecretStorage)
+
+    val storageList = ListBuffer[Secret]()
+    val secret1 = getPrivateKey25519("seed1".getBytes())
+    val secret2 = getPrivateKey25519("seed2".getBytes())
+    storageList += secret1
+    storageList += secret2
+    Mockito.when(mockedSecretStorage.getAll).thenReturn(storageList.toList)
+    Mockito.when(mockedSecretStorage.add(ArgumentMatchers.any[Secret])).thenReturn(Success(mockedSecretStorage))
+    Mockito.when(mockedSecretStorage.storeNonce(ArgumentMatchers.anyInt(), ArgumentMatchers.any[Array[Byte]])).thenReturn(Success(mockedSecretStorage))
+
+    // Prepare block ID and corresponding version
```

Comment:
These 3 lines can be removed, the field `blockId` is not used in this test

Reply:
Done

Reply:
@alsala For the future it's better to use linter